dev-node/typescript-simple/typescript-simple-8.0.6.ebuild

```diff
@@ -6,14 +6,15 @@ EAPI=7
 inherit node
 
 DESCRIPTION="Simple API to compile TypeScript code string to JavaScript. That's all!"
-HOMEPAGE="
-	https://github.com/teppeis/typescript-simple
-	https://www.npmjs.com/package/typescript-simple
-"
+HOMEPAGE="https://github.com/teppeis/typescript-simple https://www.npmjs.com/package/typescript-simple"
 
 LICENSE="MIT"
 KEYWORDS="~amd64"
 RDEPEND="
 	${NODEJS_RDEPEND}
-	|| ( node-bin/typescript dev-node/typescript )
```

Comment:
rimettimi questa riga

Reply:
Se la rimetto va in circular dep

Reply:
bisogna capire perchè

Reply:
Portage è stupido e non riesce a risolvere